lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -315,6 +318,12 @@ class HardDerivation key => SoftDerivation (key :: Depth -> * -> *) where
         -> Index 'Soft 'AddressK
         -> key 'AddressK XPub
 
+-- | Derivation of a reward account, as a type-class because different between
+-- key types (in particular, Jörmungandr vs Shelley).
+class ToChimericAccount k where
```

Comment:
this should also be reflected in `cardano-addresses` ?

Reply:
cardano-addresses doesn't actually treat with chimeric account or stake address at all at the moment. We should eventually though.

Reply:
yes, I was saying about that. We need to remember this in the context of `cardano-addresses`